src/main/tmdb/partials/awesomeSearch/AwesomeSearchController.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * AwesomeSearchController provides controller support for inline searching
+ *
+ * @module tmdb.partials.search.AwesomeSearchController
+ *
+ * @requires angular
+ * @requires config
+ * @requires TMDBAPIService
+ *
+ * @author John Gomez <jgomez@alertlogic.com>
+ *
+ * @returns instance of the AwesomeSearchController
+ *
+ * @copyright Alert Logic, Inc 2016
+ *
+ */
+
+define( [ 'angular',
+          'config/config',
+          'tmdb/services/TMDBAPIService'],
+    function( angular, config, TMDBAPIService ) {
+        "use strict";
+
+        var AwesomeSearchController = function($scope, TMDBAPIService, $timeout ) {
+            //Reference variables
+            var self = this;
+            var apiSearch = TMDBAPIService.Search();
+            var apiPerson = TMDBAPIService.Person();
+            var searchPromise;
+            self.timeWatcher = 500;
+            var limitName = 24;
+
+            var config  = angular.module("config");
+            var defaultImage = "https://simpleicon.com/wp-content/uploads/movie-1.png";
```

Comment:
I recommend copying this (or another) image into src/main/images, and referring to it via a relative link (e.g., /images/movie.png).